From daf29460408a5e0eb042b3c234c7e21a6b994cf1 Mon Sep 17 00:00:00 2001
From: Alberto Planas <aplanas@suse.com>
Date: Tue, 12 May 2020 14:16:23 +0200
Subject: [PATCH] Fix a test and some variable names (#229)

* loop: fix variable names for until_no_eval

* Fix test_core tests for fqdns errors
---
 tests/unit/grains/test_core.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/unit/grains/test_core.py b/tests/unit/grains/test_core.py
index 196dbcf83d..918a9155cb 100644
--- a/tests/unit/grains/test_core.py
+++ b/tests/unit/grains/test_core.py
@@ -1416,7 +1416,7 @@ class CoreGrainsTestCase(TestCase, LoaderModuleMockMixin):
                 with patch("salt.modules.network.log", mock_log):
                     self.assertEqual(core.fqdns(), {"fqdns": []})
                     mock_log.debug.assert_called_once()
-                    mock_log.error.assert_called()
+                    mock_log.error.assert_called_once()
 
     @patch.object(salt.utils.platform, "is_windows", MagicMock(return_value=False))
     @patch(
-- 
2.29.2


